src/pymovements/gaze/io.py

```diff
@@ -368,7 +438,8 @@ def from_asc(
         time_unit='ms',
         pixel_columns=['x_pix', 'y_pix'],
     )
-    return gaze_df, metadata
+    gaze_df._metadata = metadata  # pylint: disable=protected-access
```

Comment:
why are we calling it `_metadata` and not just `metadata`?CC @dkrako

Reply:
We're planning on removing the private attribute as soon as we found a better way to store the remaining metadata (calibrations, validations, changing sampling rate / tracked eye). See https://github.com/aeye-lab/pymovements/pull/884#issuecomment-2465477675